### PR TITLE
rbac files change JS into TS

### PR DESCRIPTION
**Pull request by 1linan**
## Description
访问角色管理模块JS语言更换成TS语言

## Checklist

- [ ] the pull request title describes what this PR does (not a vague title like `Update index.md`)
- [ ] the pull request targets the *default* branch of the repository (`develop`)
- [ ] the code follows the established code style of the repository
  - `npm run prettier:check` passes
  - `npm run eslint:check` passes
- [ ] tests are added for the changes I made (if any source code was modified)
- [ ] documentation added or updated
- [ ] I have run the project locally and verified that there are no errors

### Fixes

fixes #[issue number], fixes #[issue number]

### CODE REVIEW

File: src/pages/about/rbac/config/index.tsx [hunk 1]
```diff
@@ -133,9 +143,10 @@ function RbacConfig(props) {
       if (InvokeChainCodeMethod?.field === 'Custom') {
         getChaincodeList();
         setInvokeChaincodeCustom('Custom');
-        configValue.invokeChaincodeSubject = InvokeChainCodeMethod.custom.map((item) => item.chainCodeName);
+        configValue.invokeChaincodeSubject = InvokeChainCodeMethod.custom?.map((item) => item.chainCodeName);
       } else {
-        setInvokeChaincodeCustom();
+        //TODO:不知道你是不是想给它一个空字符串
```
Comment (Liyw-liyw) [inline src/pages/about/rbac/config/index.tsx]: 删了

File: src/pages/about/rbac/detail/index.tsx [hunk 2]
```diff
@@ -46,9 +56,10 @@ function RbacDetail(props) {
       const InvokeChainCodeMethod = policy.find((item) => item.action === 'InvokeChainCodeMethod');
       if (InvokeChainCodeMethod?.field === 'Custom') {
         setInvokeChaincodeCustom('Custom');
-        configValue.invokeChaincodeSubject = InvokeChainCodeMethod.custom.map((item) => item.chainCodeName);
+        configValue.invokeChaincodeSubject = InvokeChainCodeMethod.custom?.map((item) => item.chainCodeName);
       } else {
-        setInvokeChaincodeCustom();
+        //TODO:猜着你是想让它的值为‘’
```
Comment (Liyw-liyw) [inline src/pages/about/rbac/detail/index.tsx]: 删了

File: src/pages/about/rbac/_config.ts [hunk 3]
```diff
@@ -1,42 +1,42 @@
...
-  All = 'All',  // 可访问网络下所有资源
-  InChannel = 'InChannel',  // 可访问通道下的所有资源
-  Own = 'Own',  // 可访问自己创建的资源
-  None = 'None',  // 任何资源无法访问
-  Custom = 'Custom'  // 只能访问指定资源
+  All = 'All', // 可访问网络下所有资源
+  InChannel = 'InChannel', // 可访问通道下的所有资源
+  Own = 'Own', // 可访问自己创建的资源
+  None = 'None', // 任何资源无法访问
+  Custom = 'Custom', // 只能访问指定资源
```
Comment (willin) [inline src/pages/about/rbac/_config.ts:24]: 什么 ESLint的 规则会在末位加个 comma

**Review by willin — CHANGES_REQUESTED** [review]

File: src/pages/about/rbac/config/index.tsx [hunk 4]
```diff
@@ -19,7 +21,15 @@ breadCrumbItem.push({
   menuHref: `/`,
 });
 
-function RbacConfig(props) {
+export interface RbacConfigProps {
+  dispatch: Dispatch;
+  location: Location;
```
Comment (Liyw-liyw) [inline src/pages/about/rbac/config/index.tsx]: 使用umi的Location

**Pull request merged**